core/common/src/geometry/AdditionalTransform.ts

```diff
@@ -72,15 +74,15 @@ export class Helmert2DWithZOffset implements Helmert2DWithZOffsetProps {
     return { translationX: this.translationX, translationY: this.translationY, translationZ: this.translationZ, rotDeg: this.rotDeg, scale: this.scale };
   }
 
-  /** Compares two Helmert2DWithZOffset objects. It is a strict compare operation.
+  /** Compares two Helmert2DWithZOffset objects applying a minuscule tolerance.
    * It is useful for tests purposes only.
    *  @internal */
```

Comment:
Still need these all to be `@internal`?

Reply:
@pmconne The fact that descriptive (name, description, source ...) are strictly compared renders the equals() methods useless for any other thing than testing. What would be useful would be an equivalent() method that compares two objects to report if they are functionaly equivalent (describe the same object or transform) disregarding descriptive info. It could be planned but comparing these as equivalent() is more complex than expected and requires projection method specific code (for example standardParallel1 and standardParallel2 are interchangeable without affecting function in some cases).
I think equals() should remain internal unless you think users may want to use it for testing.

Reply:
Fair enough - thanks.

Reply:
@pmconne I had 30 minutes to think this through and the previous comment does not apply for classes with no descriptive info (such as AdditionalTransform) and the equals() can then be useful. I will review what to render public